apps/judicial-system/web/src/routes/Judge/Confirmation/Confirmation.tsx

```diff
@@ -28,29 +28,14 @@ import {
   SendNotificationMutation,
   TransitionCaseMutation,
 } from '@island.is/judicial-system-web/src/graphql'
-import { gql, useMutation, useQuery } from '@apollo/client'
+import { useMutation, useQuery } from '@apollo/client'
 import { UserContext } from '@island.is/judicial-system-web/src/shared-components/UserProvider/UserProvider'
 import { api } from '../../../services'
 import CourtRecordAccordionItem from '../../../shared-components/CourtRecordAccordionItem/CourtRecordAccordionItem'
-
-export const RequestSignatureMutation = gql`
-  mutation RequestSignatureMutation($input: RequestSignatureInput!) {
-    requestSignature(input: $input) {
-      controlCode
-      documentToken
-    }
-  }
-`
-
-export const SignatureConfirmationQuery = gql`
-  query SignatureConfirmationQuery($input: SignatureConfirmationQueryInput!) {
-    signatureConfirmation(input: $input) {
-      documentSigned
-      code
-      message
-    }
-  }
-`
+import {
+  RequestSignatureMutation,
+  SignatureConfirmationQuery,
+} from '../../../utils/mutations'
```

Comment:
Not sure it is good to have all GraphQL querys/mutations is one place. They used to be all in sharedGql.ts under the graphql folder and it got a bit crowded. Best practice says the query belongs close to its used, so perhaps in a separate file along side the tex file?

But I'll let it slide for now as it is not good to have it inside the tsx file either.